src/common/utility.ts

```diff
@@ -424,3 +424,15 @@ export const handleRowRedirect = (residence: string, current_client_country: str
 export const isLocalhost = () => !!(isBrowser() && process.env.NODE_ENV === 'development')
 
 export const isTestlink = () => !!(isBrowser() && window.location.hostname.includes('binary.sx'))
+
+export const PlatformQueryParam = () => {
```

Comment:
@mitra-deriv Is there a reason for not using the `usePlatformQueryParam` hook and creating this function instead? Is it related to server-side rendering? 🤔

Reply:
Not sure whats the issue with `gatsby query param` but when refreshing the page we have a flickering (some delay) before the sections gets hidden, so I used this function and set it true so we don’t have that flickering and it worked, if there is any better way to handle the flickering I will be happy to implement it 🙏

Reply:
@mitra-deriv I wouldn't suggest duplicating the code as we will need to maintain both places and over time each function will go in a different direction.

The flickering is probably happening because `useQueryParam` at first returns `undefined` and then gets updated with the value and causes rerendering, Maybe you can avoid this by adding an extra `state` and a `useEffect` in the `usePlatformQueryParam` hook and using a default value. (Let me know if you need help on this)

Although we already have this behavior in production, For example, if you go to https://deriv.com/?platform=p2p you can see that the header disappears after being rendered first, So maybe you can ignore it for now.

Reply:
Thanks for the suggestion I will try to fix this in a new PR and resolve this conversation once done 🙏 

cc: @habib-deriv

Reply:
Hi @farhan-nurzi-deriv , Once you applied the fix in your PR help to mention the PR number here so we can resolve this conversation.

Thank you so much for the help  🙏

Reply:
This issue is resolved here https://github.com/binary-com/deriv-com/pull/4310 🙏